akka-projection-grpc/src/test/scala/akka/projection/grpc/replication/ReplicationSettingsSpec.scala

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2009-2022 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.grpc.replication
+
+import akka.actor.testkit.typed.scaladsl.ActorTestKit
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.Behaviors
+import akka.projection.grpc.replication.scaladsl.ReplicationProjectionProvider
+import akka.projection.grpc.replication.scaladsl.ReplicationSettings
+import akka.projection.grpc.replication.javadsl.{ ReplicationSettings => JReplicationSettings }
+import akka.projection.grpc.replication.javadsl.{ ReplicationProjectionProvider => JReplicationProjectionProvider }
+import com.typesafe.config.ConfigFactory
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.concurrent.duration.DurationInt
+
+class ReplicationSettingsSpec extends AnyWordSpec with Matchers {
+
+  trait MyCommand
+
+  "The ReplicationSettings" should {
+    "Parse from config" in {
+      implicit val system: ActorSystem[Unit] = ActorSystem[Unit](
+        Behaviors.empty[Unit],
+        "parse-test",
+        ConfigFactory.parseString("""
+         // #config
+         my-replicated-entity {
+           # which of the replicas this node belongs to, should be the same
+           # across the nodes of each replica Akka cluster.
+           self-replica-id = dca
+           # max number of parallel in-flight (sent over sharding) entity updates
+           # per consumer/projection
+           parallel-updates = 8
+           # Fail the replication stream (and restart with backoff) if completing
+           # the write of a replicated event reaching the cluster takes more time
+           # than this.
+           entity-event-replication-timeout = 10s
+           replicas: [
+             {
+               # Unique identifier of the replica/datacenter, is stored in the events
+               # and can not be changed after events has been persisted.
```

Comment:
```suggestion
               # and cannot be changed after events have been persisted.
```

(`cannot` is more common than `can not`, but could be either)